### DISCUSSION TITLE
[Feature Request]: 在刮削时，能否添加合集信息

### DISCUSSION THREAD

**Discussion by Moonlight-L** [jxxghp/MoviePilot#3420] [讨论]
### 当前程序版本

1.7.4

### 运行环境

Docker

### 功能改进类型

插件

### 功能改进

您好，现在刮削的信息比较全，但导入到emby缺少合集信息，能否在媒体库刮削时，自动进行合集信息刮削

**Comment by jifengguo** [comment]
这个有需求，支持。

**Comment by kingcard1131** [comment]
Emby也可以支持自动创建合辑

**Comment by MICSAD117** [comment]
支持，这个在电影上尤其需要，比如变形金刚56部，在放映的时候很麻烦
